### ISSUE TITLE
Error on loading an existent database

### ISSUE THREAD

**Issue by vincent-lg** [evennia/evennia#1794]
#### Brief summary of issue / Description of requested feature:

When I use my old database file on the develop-py3 branch, the portal starts just fine, but the server doesn't, and I get a nasty traceback.  This doesn't happen if I start from a fresh Evennia and this didn't happen a few weeks ago.  Funny thing is develop-py3 didn't change in the meantime, so I'm really not sure what this is about.

#### Steps to reproduce the issue / Reasons for adding feature:

(Unfortunately none, it has to be something in my old database file.)

#### Error output / Expected result of feature

```
# in server.log
19-03-07 10:07:53+01 Loading d:\evennia\evennia\server\server.py...
19-03-07 10:07:54+01 Loaded.
19-03-07 10:07:54+01 twistd 18.7.0 (d:\pyenv3\scripts\python.exe 3.6.6) starting up.
19-03-07 10:07:54+01 reactor class: twisted.internet.selectreactor.SelectReactor.
19-03-07 10:07:54+01 Webserver starting on 4005
19-03-07 10:07:54+01 Unhandled Error
	Traceback (most recent call last):
	  File "d:\pyenv3\Lib\site-packages\twisted\python\log.py", line 103, in callWithLogger
	    return callWithContext({"system": lp}, func, *args, **kw)
	  File "d:\pyenv3\Lib\site-packages\twisted\python\log.py", line 86, in callWithContext
	    return context.call({ILogContext: newCtx}, func, *args, **kw)
	  File "d:\pyenv3\Lib\site-packages\twisted\python\context.py", line 122, in callWithContext
	    return self.currentContext().callWithContext(ctx, func, *args, **kw)
	  File "d:\pyenv3\Lib\site-packages\twisted\python\context.py", line 85, in callWithContext
	    return func(*args,**kw)
	--- <exception caught here> ---
	  File "d:\pyenv3\Lib\site-packages\twisted\internet\selectreactor.py", line 149, in _doReadOrWrite
	    why = getattr(selectable, method)()
	  File "d:\pyenv3\Lib\site-packages\twisted\internet\tcp.py", line 627, in doConnect
	    self._connectDone()
	  File "d:\pyenv3\Lib\site-packages\twisted\internet\tcp.py", line 656, in _connectDone
	    self.protocol.makeConnection(self)
	  File "d:\evennia\evennia\server\portal\amp.py", line 326, in makeConnection
	    amp.BinaryBoxProtocol.makeConnection(self, transport)
	  File "d:\pyenv3\Lib\site-packages\twisted\protocols\amp.py", line 2396, in makeConnection
	    self.connectionMade()
	  File "d:\evennia\evennia\server\amp_client.py", line 113, in connectionMade
	    self.factory.server.run_initial_setup()
	  File "d:\evennia\evennia\server\server.py", line 287, in run_initial_setup
	    elif int(last_initial_setup_step) >= 0:
	builtins.ValueError: invalid literal for int() with base 10: 'I-1\n.'
```

#### Extra information, such as Evennia revision/repo/branch, operating system and ideas for how to solve / implement:

* Evennia : develop-py3
* Python : 3.6.6
* System : windows

**Comment by vincent-lg** [comment]
My bad, I hadn't migrated.  Don't close it however since I get another error.  Should I open a new issue?

```
  Applying server.0002_auto_20190128_2311...Traceback (most recent call last):
  File "d:\pyenv3\Scripts\evennia_launcher.py", line 7, in <module>
    exec(compile(f.read(), __file__, 'exec'))
  File "D:\evennia\bin\windows\evennia_launcher.py", line 17, in <module>
    main()
  File "d:\evennia\evennia\server\evennia_launcher.py", line 2159, in main
    django.core.management.call_command(*args, **kwargs)
  File "d:\pyenv3\Lib\site-packages\django\core\management\__init__.py", line 131, in call_command
    return command.execute(*args, **defaults)
  File "d:\pyenv3\Lib\site-packages\django\core\management\base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "d:\pyenv3\Lib\site-packages\django\core\management\commands\migrate.py", line 204, in handle
    fake_initial=fake_initial,
  File "d:\pyenv3\Lib\site-packages\django\db\migrations\executor.py", line 115, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "d:\pyenv3\Lib\site-packages\django\db\migrations\executor.py", line 145, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "d:\pyenv3\Lib\site-packages\django\db\migrations\executor.py", line 244, in apply_migration
    state = migration.apply(state, schema_editor)
  File "d:\pyenv3\Lib\site-packages\django\db\migrations\migration.py", line 129, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "d:\pyenv3\Lib\site-packages\django\db\migrations\operations\special.py", line 193, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "d:\evennia\evennia\server\migrations\0002_auto_20190128_2311.py", line 17, in forwards
    value = loads(to_bytes(conf.db_value), encoding='bytes')   # py2->py3 byte encoding
_pickle.UnpicklingError: pickle data was truncated
```

**Comment by vincent-lg** [comment]
Okay you can close, it was a double error on me.  Don't start Evennia before migrating since it will update your DB even in case of errors and the next migrations will break.
